packages/react-instantsearch/src/connectors/connectSearchBox.js

```diff
@@ -10,6 +10,9 @@ function getCurrentRefinement(props, searchState) {
   if (typeof searchState[id] !== 'undefined') {
     return searchState[id];
   }
+  if (typeof props.defaultRefinement === 'string') {
```

Comment:
connectSearchBox has no proptypes defined, I will add them. And then only do an `!== undefined` test, works?